desktop/public_desktop/preload.js

```diff
@@ -52,6 +52,10 @@ function fflagOverrides() {
 		const kv = fflags[ i ].split( ':' );
 		payload[ kv[ 0 ] ] = kv[ 1 ] === 'true' ? true : false;
 	}
+
+	// Manual override of feature flags for features not available in older app versions
+	payload[ 'sign-in-with-apple' ] = true;
```

Comment:
Suggestion: Perhaps we can add the feature flags to the payload _first_, and then allow the environment variable to override them with `Object.assign` or the spread operator. There may be cases where we want to disable certain flags to assess behavior, so we should probably allow for overrides of existing flags.

Reply:
Good call. I have this updated now!